### PR TITLE
[AutoMM] PDF document classification tutorial

### PR DESCRIPTION
**Pull request by cheungdaven**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/multimodal/document/pdf_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    "from IPython.display import IFrame\n",
+    "IFrame(\"https://automl-mm-bench.s3.amazonaws.com/doc_classification/historical_1.pdf\", width=400, height=500)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, this document is an America's historical document in PDF format. \n",
```
Comment (suzhoum) [inline docs/tutorials/multimodal/document/pdf_classification.ipynb:81]: I think the PDF is not displaying in the tutorial.

File: docs/tutorials/multimodal/document/pdf_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Other Examples\n",
+    "\n",
+    "You may go to [AutoMM Examples](https://github.com/autogluon/autogluon/tree/master/examples/automm) to explore other examples about AutoMM.\n",
+    "\n",
+    "## Customization\n",
+    "To learn how to customize AutoMM, please refer to :ref:`sec_automm_customization`.\n"
```
Comment (suzhoum) [inline docs/tutorials/multimodal/document/pdf_classification.ipynb:222]: You might need this to make the hyperlink work: `[Customize AutoMM](../advanced_topics/customization.ipynb)`

File: docs/tutorials/multimodal/document/pdf_classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    "[![Open In SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/autogluon/autogluon/blob/master/docs/tutorials/multimodal/document/pdf_classification.ipynb)\n",
+    "\n",
+    "PDF comes short from Portable Document Format and is one of the most popular document formats.\n",
+    "We can find PDFs everywhere, from personal resumes to business contracts, and from commercial brochures to government documents. \n",
+    "The list can be endless. \n",
+    "PDF is highly praised for its portability. \n",
+    "There's no worry about the receiver being unable to view the document or see an imperfect version regardless of their operating system and device models.\n",
+    "\n",
+    "Using AutoMM, you can handle and build machine learning models on PDF documents just like working on other modalities such as text and images, without bothering about PDFs processing. \n",
+    "In this tutorial, we will introduce how to sort PDF documents automatically with AutoMM using document foundation models. Let’s get started!"
```
Comment (zhiqiangdon) [inline docs/tutorials/multimodal/document/pdf_classification.ipynb:20]: `sort PDF` = `classify PDF`? Users may get confused.

File: docs/tutorials/multimodal/document/pdf_classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from autogluon.multimodal import MultiModalPredictor\n",
+    "\n",
+    "predictor = MultiModalPredictor(label=\"label\", verbosity=5)\n",
```
Comment (zhiqiangdon) [inline docs/tutorials/multimodal/document/pdf_classification.ipynb:123]: Using `verbosity=5` outputs some debugging info, which should be avoided in tutorials.

File: docs/tutorials/multimodal/document/pdf_classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    "predictor.fit(\n",
```
[the remark this answers is not in the document]
Comment (cheungdaven) [reply docs/tutorials/multimodal/document/pdf_classification.ipynb:124]: solved.

**Review by suzhoum — APPROVED** [review]
LGTM!

**Review by zhiqiangdon — APPROVED** [review]
LGTM

**Pull request merged**